core/src/main/java/io/parsingdata/metal/expression/value/reference/First.java

```diff
@@ -16,35 +16,30 @@
 
 package io.parsingdata.metal.expression.value.reference;
 
-import static io.parsingdata.metal.Util.checkNotNull;
-
 import io.parsingdata.metal.data.Environment;
-import io.parsingdata.metal.data.ParseValueList;
+import io.parsingdata.metal.data.OptionalValueList;
 import io.parsingdata.metal.encoding.Encoding;
-import io.parsingdata.metal.expression.value.OptionalValue;
 import io.parsingdata.metal.expression.value.ValueExpression;
 
+import static io.parsingdata.metal.Util.checkNotNull;
+
 public class First implements ValueExpression {
 
-    private final String _name;
+    private final ValueExpression _op;
```

Comment:
(In other new/changed code you had already used field names without underscore, but not here?)